src/components/[guild]/activity/ActivityLogFiltersBar/components/Suggestions.tsx

```diff
@@ -0,0 +1,122 @@
+import { Stack, Text } from "@chakra-ui/react"
+import * as combobox from "@zag-js/combobox"
+import { HTMLAttributes, useMemo } from "react"
+import { ActivityLogType, useActivityLog } from "../../ActivityLogContext"
+import { Filter, SupportedQueryParam } from "./ActivityLogFiltersContext"
+import Suggestion from "./Suggestion"
+
+interface Props {
+  contentProps: HTMLAttributes<HTMLElement>
+  activityLogType: ActivityLogType
+  activeFilters: Filter[]
```

Comment:
We could omit this prop & use it from the `useActivityLogFilters` hook instead

Reply:
nice catch, thanks!